app/models/organization.rb

```diff
@@ -39,7 +39,10 @@ class Organization < ApplicationRecord
   scope :order_by_oldest, ->(_context = nil) { order(created_at: :asc) }
   scope :order_by_title,  ->(_context = nil) { order(:title) }
 
-  scope :search_by_title, ->(query) { where("title ILIKE ?", "%#{query}%") }
+  scope :search_by_title, ->(query) { where("organizations.title ILIKE ?", "%#{query}%") }
```

Comment:
This was failing tests due to being an ambiguous field so I added in the explicit reference to the `organizations` table.